packages/shared/src/cli/git.tag.ts

```diff
@@ -0,0 +1,6 @@
+import { execFileSync } from 'child_process';
+
+export const GitTag = {
+    version: execFileSync('git', ['describe', '--tags', '--always', '--match', 'v*']).toString().trim(),
+    hash: execFileSync('git', ['rev-parse', 'HEAD']).toString().trim(),
```

Comment:
any idea what this returns inside of CI ?

Reply:
The build passes so I assume it is a valid version and hash

Reply:
Is it the expected value?

Reply:
I assume so because the test passes.